dso/model.py

```diff
@@ -566,6 +676,8 @@ class ArtefactMetadata:
         | OsID
         | CustomRescoring
         | ComplianceSnapshot
+        | FalcoEventGroup
```

Comment:
I think it might be more adequate to group debug/non-debug events under one common artefact-metadata-type (you can of course have different subtypes handled within falco's odg-extension) - from ODG's pov, I think this discrimination is an unsignificant implementation detail.

in fact, you already use the same label for github-tracking-issues (which kind of shows you seem to see it the same way)

Reply:
adjusted, please PTAL